frontend/tet/admin/src/pages/_document.tsx

```diff
@@ -0,0 +1,16 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document';
```

Comment:
There is `ServerDocment` in shared code which perhaps could be used?... just the portal part should rendered only optionally, if some new property (ie `portal:boolean`) is given.

Reply:
Ok, thanks! I missed the shared option. I'll check if that could be used.